Libraries/Components/ActivityIndicator/ActivityIndicator.js

```diff
@@ -29,9 +29,10 @@ type IndicatorSize = number | 'small' | 'large';
 
 type IOSProps = $ReadOnly<{|
   /**
-   * Whether the indicator should hide when not animating (true by default).
+   * Whether the indicator should hide when not animating.
    *
-   * See https://reactnative.dev/docs/activityindicator.html#hideswhenstopped
+   * @see https://reactnative.dev/docs/activityindicator.html#hideswhenstopped
```

Comment:
@ani4aniket What's the purpose of the `@see` annotations? If it's just provide the anchor link for the header, I believe that can be automatically generated and does not need to be specified in docs?

Reply:
@jevakallio, I have just written @see just for the uniformity as another URL is mention in inheritedProps.
so just tried to make it as an annotation for URLs
`@inheritsProps View {@see https://reactnative.dev/docs/view.html#props}`

Reply:
Ah, I **see**! 😄 🙄

We don't need to specify these anchor links manually, Docusaurus will do that for us. The link format is `http://generate.page/url.html#generated-slug-from-heading`, where
- The reference URL of this page is always going to be the full url to the generated page, wherever its hosted, 
- The anchor section after `#` is the content of the heading

So, even if we wanted to, we *can't* specify these URLs ourselves.

The `@see` annotation is useful when wanting to refer **to** other sections and articles. So, if for example the documentation reader would benefit from also reading another section on another page, then you could use:

```
@see https://reactnative.dev/docs/view.html#props
```

Reply:
Cool, thanks for the clarification.